packages/core/src/utils/breadcrumbsUtils.ts

```diff
@@ -0,0 +1,17 @@
+import type { SeverityLevel } from '@sentry/types';
+
+/**
+ * Determine a breadcrumb's log level based on the response status code
+ * @param statusCode
+ */
+export function getBreadcrumbLogLevel(statusCode: number | undefined): SeverityLevel {
+  if (statusCode === undefined) {
+    return 'info';
+  } else if (statusCode >= 400 && statusCode < 500) {
+    return 'warning';
+  } else if (statusCode >= 500) {
+    return 'error';
+  } else {
+    return 'info';
```

Comment:
For the `undefined` and other cases, the fallback level is `info`. So, all the test cases for breadcrumbs in `browser-integration-tests` must be updated. 

Is this behaviour needed? E.g., success request breadcrumb will have an additional property `level: 'info'`

Reply:
Currently many tests are failing b/c of the default property. Maybe we shouldn't set it if the response has a non-error status code. Otherwise, it's a handful to fix those tests (though it's simply just adding the property). 

Any thoughts?

Reply:
For now, i've left out the default value (`info`) to not break the many other tests. But other sdks use a default value: https://github.com/getsentry/sentry-php/pull/1773/commits/d12482aebd218ce8f20d55f4fc276940b24a8769. So we should consider it.